src/bitmessagemain.py

```diff
@@ -405,12 +406,22 @@ def getApiAddress(self):
         port = BMConfigParser().getint('bitmessagesettings', 'apiport')
         return {'address':address,'port':port}
 
+    
+def emojiLogg():
+    from debug import logger
+    reload(sys)  
+    sys.setdefaultencoding(  'utf8')
+    logger.warning(str(     u'exc:äüöß   🤷 🤷  without exception  😃                         '.encode("UTF-8")   )          )
+    logger.warning(          ' äüß      🤷 🤷 🤷   -- just a test emoji warning to demo this  ')
```

Comment:
Please get rid of the logger.warning or replace it with something more neutral, like "logger encoding reset", and it should be .info or .debug, not .warning.